lib/api.c

```diff
@@ -20,6 +20,13 @@ EXP ryzen_access CALL init_ryzenadj()
 	if (family == FAM_UNKNOWN)
 		return NULL;
 
+#ifdef __linux__
+	if (getuid()) {
+		printf("Please, run as root!\n");
```

Comment:
On Linux it is possible to grant devfs permission for regular users with CAP.
So this is not correct.

Reply:
All right. I didn't know this detail :ok_hand: 

Thank you.